core/tee/tee_svc_cryp.c

```diff
@@ -2989,6 +3052,7 @@ TEE_Result tee_svc_asymm_verify(uint32_t state, const TEE_Attribute *params,
 	struct tee_obj *o;
 	size_t hash_size;
 	int salt_len;
+	TEE_Attribute *params = NULL;
```

Comment:
We don't know the value of num_params in advance and we don't have alloca(), I don't see how it can be achieved.